servicetalk-encoding-api/src/main/java/io/servicetalk/encoding/api/AbstractZipContentCodec.java

```diff
@@ -589,10 +571,10 @@ private void verifyCrc(Buffer in) throws IOException {
     }
 
     static class SwappableBufferOutputStream extends OutputStream {
+        @Nullable
```

Comment:
if you don't wan't to worry about null you can initialize to `EmptyBuffer.EMPTY_BUFFER`

Reply:
I did consider that, but spot-buts starts complaining with ` RV_RETURN_VALUE_IGNORED_NO_SIDE_EFFECT: Return value of method without side effect is ignored` and I didn't won't to suppress it.

Reply:
hum this seems like a general problem that we may want to address, but independent issue